src/screens/Settings.js

```diff
@@ -126,9 +126,13 @@ class Settings extends React.Component {
 
   render() {
     const serverURL = hathorLib.helpers.getServerURL();
+    const now = new Date();
     return (
       <div className="content-wrapper settings">
         <BackButton {...this.props} />
+        <div>
+          <p><strong>{t`Date and time:`}</strong> {now.toString()}</p>
```

Comment:
Should we put the `timestamp` in a "!" badge?

Reply:
Done. Added the badge with the timestamp as title